artisan.md

```diff
@@ -170,3 +170,7 @@ In this example, the `foo` command will be run every minute if it is not already
 #### Ping A Given URL After The Job Runs
 
 	$schedule->command('foo')->thenPing($url);
+
+Using the `thenPing($url)` feature require that the Guzzle 4 HTTP library be installed into your application. You can add Guzzle 4 to your project by adding the following line to your `composer.json` file:
```

Comment:
Okay, will update here *and* in the `mail.md`, because that's where I copied this from. ;)